src/main/groovy/wooga/gradle/unity/traits/UnitySpec.groovy

```diff
@@ -44,6 +46,10 @@ trait UnitySpec extends BaseSpec {
     void setProjectDirectory(Provider<Directory> value) {
         projectDirectory.set(value)
     }
+    void setProjectDirectory(String path) {
+        projectDirectory.set(layout.projectDirectory.dir(path))
+    }
+
```

Comment:
I thought we decided not to add too many overloads for the setters. And in this case, because we have a few overloads we wanted to keep, was to use `File` over `String`. Because in the build.gradle file one can express this quite nicely with:
```groovy
unity.projectDirectory = file("path/to/project")
```

Which is cleaner than allowing any String to be passed to the setter.

Reply:
Very well.